nservicebus/sqlserver/multiple-databases.md

```diff
@@ -1,21 +1,66 @@
 
-NOTE: If the destination endpoint uses a different database or server instance, sending a message to it might cause the transaction to escalate to a distributed transaction which may not be desirable. Using the [Outbox](/nservicebus/outbox/ feature, DTC escalations can be avoided as long as the both the endpoint's Outbox and business data share the same database.
+WARNING: SQL Server transport does not support store-and-forward mechanism on a transport level, so there is no dead-letter queue. It means that in *multi-catalog* and *multi-database* modes if a remote endpoint's infrastructure (e.g. DTC or SQL Server instance) is down, then messages can't be delivered to it. That makes this particular endpoint, and all endpoints depending on it, unavailable. That problem can be addressed by using [Outbox](/nservicebus/outbox/) feature, as explained in [*Multi-catalog* and *multi-instance* with store-and-forward](#multi-catalog-and-multi-instance-with-store-and-forward) section. 
 
+NOTE: Sending messages between endpoints in *multi-catalog* and *multi-instance* modes causes the transaction to escalate to a distributed transaction. In the *multi-catalog* mode the escalation might be avoided if [Outbox](/nservicebus/outbox/) feature is turned on, and Outbox data and business data use the same connection string (storing data in a single database might not be sufficient condition as described in [Entity Framework caveats](/nservicebus/sqlserver/configuration.md#entity-framework-caveats) section). Storing Outbox data in the same database as business data in practice means choosing NHibernate persistence, and configuring it to use the appropriate connection string.
+
+# Modes overview
+
+## Default
+
+- Supported in Versions 1.0 and higher
+- Simple configuration and setup
+- Doesn't require Distributed Transaction Coordinator (MS DTC)
+- The snapshot (backup) of the entire system state is done by backing up a single database (especially useful if business data is also stored in the same database)
+- Can be monitored with ServiceControl
+
+## Multi-schema
+
+- Supported in Versions 2.1 and higher
+- Simple configuration and setup
+- Doesn't require Distributed Transaction Coordinator (MS DTC)
+- The snapshot (backup) of the entire system state is done by backing up a single database (especially useful if business data is also stored in the same database)
+- A fine-grained security control on a database schema level
+- Can be monitored with ServiceControl
+
+## Multi-catalog
+
+- Supported in Versions 2.1 and higher
+- Requires Distributed Transaction Coordinator by default
+- Escalation to distributed transactions can be avoided by using Outbox and storing business data in the same database as Outbox data
+- Can be monitored with ServiceControl
+
+## Multi-instance
+
+- Supported in Versions 2.1 to 3.x, won't be supported in Versions 4.0 and higher
+- Requires Distributed Transaction Coordinator, even when using Outbox
```

Comment:
This is the same as for multi-catalog. We can avoid this if `errors` are on the same instance.